galette/templates/default/pages/preferences.html.twig

```diff
@@ -884,21 +899,25 @@
                         }
                     }
                 });
-                /*$('#smtp, #gmail').checkbox({
-                    onChecked: function() {
-                        console.log($(this).closest('#smtp_parameters'));
-                        //$('#smtp_parameters, #smtp_auth').toggleClass('displaynone');
-                    }
-                });*/
 
                 $('.pointing.menu .item').tab({
                     onVisible: function(tabPath) {
                         document.getElementById('tab').value = tabPath;
                     }
                 });
 
-                $('#pref_bool_publicpages').change(function(){
-                    $('#publicpages_visibility').toggleClass('displaynone');
+                let _publicpages_status = document.getElementById('pref_bool_publicpages');
+                let _publicpages_visibility = document.getElementById('publicpages_visibility');
+                _publicpages_status.addEventListener('change', function () {
+                    _publicpages_visibility.classList.toggle('displaynone');
+                    return;
+                });
```

Comment:
Does that changes something or does that fix an issue? (real question - dont' change ;))

Reply:
It's the equivalent with javascript only.
I tend not to use jquery when possible.
You can revert if you want :wink:

Reply:
No, it's OK, I was just wondering ;)